lib/components/image_banner.dart

```diff
@@ -22,18 +23,17 @@ class ImageBanner extends HookConsumerWidget {
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     return LayoutBuilder(builder: (context, constraints) {
-      if (constraints.maxWidth > 600) {
-        var desktopImageBanner = DesktopImageBanner(
+      if (Platform.isAndroid || Platform.isIOS) {
```

Comment:
Yes, they are called Mobile and Desktop but they are more like Portrait and Landscape Banners. The sizing corresponds to the layout breakpoints I set. 
Limiting it to platforms would mean Tablets always get a protrait mode banner even if in landscape mode and desktops always get a landscape mode banner even if the window is vertical

Reply:
what do you see as the important design elements in this choice?  for me this is acceptable because the gradient is nice when it shrinks very wide viewports so that I don't have to scan back and forth to look at a banner.  
and now, of course, they require different handling as far as MouseArea and Listener are concerned.  It would be nice to restore the gradient on large landscape tablets, I agree. how would you feel about a synthesis of all those princples and all of these ones?

Reply:
other than my note below I think this is resolved.